packages/core/src/registry.ts

```diff
@@ -267,16 +283,35 @@ export class Registry implements IRegistry {
     return this;
   }
 
-//////////////////////////////////////////
+  //////////////////////////////////////////
 
   // todo, refactor, this is a mess, see where-used, a lot of the code should be in this method instead
   private parsePrivate(input: IObject) {
     if (input instanceof ABAPObject) {
-      const result = input.parse(this.getConfig().getVersion(), this.getConfig().getSyntaxSetttings().globalMacros);
+      const config = this.getConfig();
+      const result = input.parse(config.getVersion(), config.getSyntaxSetttings().globalMacros);
       ParsingPerformance.push(input, result);
     }
   }
 
+  private withoutGloballyExcludedFiles(issues: Issue[], globalExclude: string[]): Issue[] {
```

Comment:
then this method can be removed?

Reply:
did it just as you commented

Reply:
okay, I were looking at an old version 👍